addons/crm_claim/data/crm_claim_data.xml

```diff
@@ -1,63 +1,56 @@
-<?xml version="1.0"?>
-<openerp>
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
     <data noupdate="1">
 
-        <!--
-        Claims Categories
-        -->
+        <!--Claims Categories-->
         
-        <record model="crm.claim.category" id="categ_claim1">
+        <record id="categ_claim1" model="crm.claim.category">
             <field name="name">Factual Claims</field>
             <field name="team_id" ref="sales_team.team_sales_department"/>
             </record>
         
-        <record model="crm.claim.category" id="categ_claim2">
+        <record id="categ_claim2" model="crm.claim.category">
             <field name="name">Value Claims</field>
             <field name="team_id" ref="sales_team.team_sales_department"/>
              </record>
         
-        <record model="crm.claim.category" id="categ_claim3">
+        <record id="categ_claim3" model="crm.claim.category">
             <field name="name">Policy Claims</field>
             <field name="team_id" ref="sales_team.team_sales_department"/>
         </record>
     
-        <!--
-        Case Category2
-        -->
+        <!--Case Category2-->
         
-        <record model="utm.campaign" id="claim_source1">
+        <record id="claim_source1" model="utm.campaign">
```

Comment:
Is really need diff ?
![gifd](https://cloud.githubusercontent.com/assets/14940179/12152071/87665390-b46f-11e5-846b-bf05bfc03992.gif)